frontend/public/components/broker-management.tsx

```diff
@@ -1,27 +1,32 @@
 import * as React from 'react';
+import i18next from 'i18next';
+import { useTranslation } from 'react-i18next';
 import { HorizontalNav, PageHeading } from './utils';
 import { ClusterServiceBrokerPage } from './cluster-service-broker';
 import { ClusterServiceClassPage } from './cluster-service-class';
 
 const pages = [
   {
     href: '',
-    name: 'Service Brokers',
+    name: i18next.t('broker-management~Service brokers'),
```

Comment:
This component is deprecated and doesn't need to be updated. You can remove this file from the PR.

Reply:
Reverted the change of broker-management